### ISSUE TITLE
Software crashing due to USBD480_lib.dll

### ISSUE THREAD

**Issue by redion1992** [SHWotever/SimHub#410]
The software can crash silently, even when out of game. No mention is listed in the logs. Based on comparing the log timing to the Windows error report, the software crashed roughly 54 minutes after starting.

The maintenance data from Windows gives the following:
```
Description
Faulting Application Path:	C:\Program Files (x86)\SimHub\SimHubWPF.exe

Problem signature
Problem Event Name:	BEX
Application Name:	SimHubWPF.exe
Application Version:	1.0.0.0
Application Timestamp:	5d7fffee
Fault Module Name:	USBD480_lib.dll
Fault Module Version:	0.0.0.0
Fault Module Timestamp:	59f5d74e
Exception Offset:	00004934
Exception Code:	c0000409
Exception Data:	00000000
OS Version:	10.0.18362.2.0.0.256.48
Locale ID:	3081
Additional Information 1:	09ba
Additional Information 2:	09ba2ce28ffdcbd78d202d0967041cdc
Additional Information 3:	8f79
Additional Information 4:	8f796768ffdc937f03fc3c7ad6db8998
```

I have just disabled USB480D support, and am waiting to see if that fixes the issue. If this is the case, I would suggest that such support is disabled by default rather than enabled on blank configs.

**Comment by SHWotever** [comment]
Hi! Thanks for the info! 
I've already seen some random crashes related to the USBD480 when it was not available on the system, I've made the modifications to make sure it won't call at all the usbd480 detection routines when it's disabled and it will be disabled by default on fresh installs.

It will be available on the next release.

**Comment by SHWotever** [comment]
Released with 6.7.8, thanks again for the feedback !
